### DISCUSSION TITLE
New Custom domain added not working

### DISCUSSION THREAD

**Discussion by swetanks** [vercel/community#5311] [Help] [answered]
### Summary

I have setup the custom domain **kutumbh.com** on vercel. The build is perfectly deployed. The preview link from vercel is able to display the static website.
Verified all DNS checker, both A record and CNAME records are pointing to vercel
The website is still not accessible.

### Example

https://www.kutumbh.com/

### Steps to Reproduce

1. Custom domain setup as mentioned
2. Open URL www.kutumbh.com

**Comment by PranavArya37** [comment]
I am also facing the same problem but your URL works fine now. Can you please tell me what you have done or if it has happened automatically and if it happened automatically then how much time it took


Mine is showing something like this-
![image](https://github.com/vercel/community/assets/43077165/5f27ee53-26f1-4380-8f12-8cf1d136b7de)

**Reply by swetanks** [reply, marked as answer]
It happened automatically. no change done.
